layouts/shortcodes/readfile.md

```diff
@@ -1,11 +1,119 @@
-{{ if eq (.Get "relative") "true" }}
+{{/*
+
+Overrides Hugo's readFile (https://gohugo.io/functions/readfile/).
+
+Use this `readfile` shortcode to include the contents of a file into another file.
+Specify the `readfile` shortcode in a specific location within the "parent"
+file to "include" content from the source file.
+
+Default behavior is to obtain source files in a directory that is relative to
+the parent file. See details below.
+
+IMPORTANT: You cannot nest shortcodes. When the Hugo build runs on a file,
+any shortcodes within that file are processed. For example, if you use `readFile`
+to include a source file, and that source file contains another shortcode,
+none of those "nested shortcodes" are processed. Nesting a shortcode within another
+shortcode results in incorrectly rendered content
+(ie. `{{% the short code syntax shows in the HTML of the parent file %}}`).
+
+Syntax:
```

Comment:
This doesn't render well in markdown, maybe just an FYI: https://github.com/knative/website/blob/c0e202237ffa93b299f10207e67687a4768061b3/layouts/shortcodes/readfile.md

Reply:
fixed

Reply:
fixed